camera_frame_name: "camera"
frequency: 30